src/Curiosity/curiosity.tsx

```diff
@@ -0,0 +1,35 @@
+  
+
+import React from 'react';
+import {useState, useEffect} from "react";
+import {BrowserRouter as Router, Switch, Route, Link} from 'react-router-dom';
+import './curiosity.scss';
+
+function Curiosity() {  
+    const [myPhotos, setMyPhotos] = useState(null);
```

Comment:
I think we can call this something more descriptive than myPhotos, like "roverPhotoData" or even better "curiosityPhotoData"